src/EPR.Calculator.Service.Function/Services/CalculatorRunService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using EPR.Calculator.Service.Common;
+using EPR.Calculator.Service.Function.Interface;
+using System;
+using EPR.Calculator.Service.Function.Constants;
+
+
+namespace EPR.Calculator.Service.Function.Services
+{
+    public class CalculatorRunService : ICalculatorRunService
+    {
+             
+        public void startProcess(CalculatorRunParameter calculatorRunParameter)
+        {
+           var environmentConfiguration =   GetConfiguration(true);
```

Comment:
Why are we using true as a parameter? Do we need a variable for it?

Reply:
created a variable for the same.